internal/copy_handler.go

```diff
@@ -0,0 +1,141 @@
+package internal
+
+import (
+	"path"
+	"strings"
+
+	"github.com/wal-g/storages/storage"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/utility"
+)
+
+type copyingInfo struct {
+	object storage.Object
+	from   storage.Folder
+	to     storage.Folder
+}
+
+// HandleCopy copy specific or all backups from one storage to another
+func HandleCopy(fromConfigFile string, toConfigFile string, backupName string, withoutHistory bool) {
+	var from, fromError = ConfigureFolderFromConfig(fromConfigFile)
+	var to, toError = ConfigureFolderFromConfig(toConfigFile)
+	if fromError != nil || toError != nil {
+		return
+	}
+	var infos, err = getObjectsToCopy(backupName, from, to, withoutHistory)
+	if err != nil {
+		tracelog.ErrorLogger.FatalError(err)
```

Comment:
Go way: handle error on the top level.
It's top-level and if it was Fatal you should exit. Use FatalOnError here